main.go

```diff
@@ -80,6 +82,9 @@ func newArgParser() (*flags.Parser, *options) {
 			"Multiple patches may be provided to be applied in-order. " +
 			"If the flag is omitted, a patch will be read from stdin."
 
+	parser.FindOptionByLongName("lint").Description =
+		"Turn on lint flag to output patch matches to stdout instead of modifying the files"
```

Comment:
This is a bit awkward. It effectively says that if I use the `--lint` flag, I turn the lint flag on.
Maybe, 

"Turn on linter mode that writes a diff to stdout instead of modifying the files."

Seeing as this mode prints a diff, should we call this "--diff" instead?